interop-cats/shared/src/main/scala/zio/interop/Schedule.scala

```diff
@@ -159,7 +159,7 @@ final class Schedule[F[+_], -In, +Out] private (private[Schedule] val underlying
   def combineWith[In1 <: In, Out2](
```

Comment:
In ZIO we deprecated `combineWith` and added the `unionWith` and `intersectWith` operators. I would suggest we do the same here to maintain a consistent API.

Reply:
Thank you for the review. Deprecated `combineWith` in favour of `intersectWith ` and added `unionWith` as well.